libs/net/dal/Migrations/1.0.9/Up/PreUp/01-action.sql

```diff
@@ -0,0 +1,7 @@
+DO $$
+BEGIN
+
+DELETE FROM public."action"
```

Comment:
We don't need the Front Page action.  Front Pages are specified through the Product.